server/gitrest/src/routes/git/refs.ts

```diff
@@ -136,7 +145,16 @@ function handleResponse(resultP: Promise<any>, response: Response, successCode:
  * Simple method to convert from a path id to the git reference ID
  */
 function getRefId(id): string {
-    return `refs/${id}`;
+    return `refs/heads/${id}`;
+}
+
+function getReadParams(params): IGetRefParamsExternal {
```

Comment:
(nit) `: IGetRefParamsExternal | undefined` would be more accurate.

Alternatively, returning a default config instead of undefined would work (e.g. `{ config: { enabled: false } }` or `{}`)